daemon/src/engine/client/cg_msgdef.h

```diff
@@ -719,6 +721,9 @@ using CGameTextInptEvent = IPC::SyncMessage<
 using CGameFocusEventMsg = IPC::SyncMessage<
 	IPC::Message<IPC::Id<VM::QVM, CG_FOCUS_EVENT>, bool>
 >;
+using CGameRecvMessageMsg = IPC::SyncMessage<
+	IPC::Message<IPC::Id<VM::QVM, CG_RECV_MESSAGE>, IPC::SharedMemory, size_t, int>
```

Comment:
Does a command buffer involve copying memory around? Ideally shared memory would be created once and reused